pom.xml

```diff
@@ -100,138 +100,145 @@
 
   <properties>
     <maven.compiler.release>8</maven.compiler.release>
-    <auto-value.version>1.11.0</auto-value.version>
-    <avro.version>1.11.3</avro.version>
-    <bouncycastle.version>1.78.1</bouncycastle.version>
-    <!--Ensure Beam SDK compatibility-->
-    <!-- https://github.com/apache/beam/blob/release-2.59.0/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L586 -->
-    <beam.version>2.59.0</beam.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+    <!-- apache beam BOM -->
+    <!-- https://github.com/apache/beam/blob/release-2.60.0/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L586 -->
+    <beam.version>2.60.0</beam.version>
+    <!-- versions from beam -->
+    <auto-value.version>1.9</auto-value.version>
+    <checker-qual.version>3.42.0</checker-qual.version>
+    <commons-codec.version>1.17.1</commons-codec.version> <!-- libraries-bom version -->
+    <commons-compress.version>1.26.2</commons-compress.version>
+    <errorprone.version>2.10.0</errorprone.version>
+    <guava.version>33.1.0-jre</guava.version>
+    <hamcrest.version>2.1</hamcrest.version>
+    <httpclient.version>4.5.13</httpclient.version>
+    <httpcore.version>4.4.14</httpcore.version>
+    <jackson.version>2.15.4</jackson.version>
+    <joda-time.version>2.10.14</joda-time.version>
+    <netty.version>4.1.100.Final</netty.version>
+    <slf4j.version>1.7.30</slf4j.version>
+    <threetenbp.version>1.6.8</threetenbp.version>
+    <zstd-jni.version>1.5.6-3</zstd-jni.version>
+
+    <!-- GCP BOM -->
     <!-- https://github.com/googleapis/java-cloud-bom/releases -->
     <!-- https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/26.45.0/index.html -->
     <google-cloud-libraries-bom.version>26.45.0</google-cloud-libraries-bom.version>
-    <guava.version>33.3.0-jre</guava.version>
-    <hamcrest.version>3.0</hamcrest.version>
+    <!-- versions from GCP bom -->
+    <opencensus.version>0.31.1</opencensus.version>
+
+    <!-- other deps -->
+    <avro.version>1.11.4</avro.version>
```

Comment:
Shouldn't we put this on on the beam BOM? `2.60.0` uses `1.11.3`

Reply:
Yes, done